Makefile

```diff
@@ -476,13 +476,21 @@ generate-addons: fetch-calico-manifests ## Generate metric-server, calico calico
 
 # When updating this, make sure to also update the Windows image version in templates/addons/windows/calico.
 CALICO_VERSION := v3.23.0
+CALICO_RELEASES := $(ARTIFACTS)/calico
+CALICO_MANIFEST_PATH := release-$(CALICO_VERSION)/manifests/calico-vxlan.yaml
+CALICO_MANIFEST_IPV6_PATH := release-$(CALICO_VERSION)/manifests/calico-policy-only.yaml
+CALICO_MANIFEST := $(CALICO_RELEASES)/$(CALICO_MANIFEST_PATH)
+CALICO_MANIFEST_IPV6 := $(CALICO_RELEASES)/$(CALICO_MANIFEST_IPV6_PATH)
 
 .PHONY: fetch-calico-manifests
-fetch-calico-manifests: ## Get Calico release manifests and unzip them.
+fetch-calico-manifests: $(CALICO_MANIFEST) $(CALICO_MANIFEST_IPV6) ## Get Calico release manifests and unzip them.
```

Comment:
I'll actually tune it a bit so it plays well when running targets in parallel.

Reply:
Done. As an alternative, I could make one target for regular manifests and then IPv6 target could depend on this one, however that seem a bit more hacky to me.